include/translatable-object.php

```diff
@@ -452,4 +452,46 @@ protected function get_objects_with_no_lang_sql( $language_ids, $limit ) {
 			$limit >= 1 ? sprintf( 'LIMIT %d', $limit ) : ''
 		);
 	}
+
+	/**
+	 * Assigns a language to object in mass.
+	 *
+	 * @since 1.2
+	 * @since 3.4 Moved from PLL_Admin_Model class.
+	 *
+	 * @param int[]               $ids  Array of post ids or term ids.
+	 * @param PLL_Language|string $lang Language to assign to the posts or terms.
+	 * @return void
+	 */
+	public function set_language_in_mass( $ids, $lang ) {
+		global $wpdb;
+
+		$lang = $this->get_language( $lang );
+
+		if ( empty( $lang ) ) {
+			return;
+		}
+
+		$tax_language = $this->get_tax_language();
+		$tt_id        = $lang->get_tax_prop( $tax_language, 'term_taxonomy_id' );
```

Comment:
Not neccessary to use the getter in the class itself:
```suggestion
		$tt_id = $lang->get_tax_prop( $this->tax_language, 'term_taxonomy_id' );
```

Reply:
Done in d5f6ee7d27ba75515801f5154db53476927b3b96